ThirdPartyLicenses.txt

```diff
@@ -0,0 +1,52 @@
+Third Party Notices
+
+This Microsoft Open Source project incorporates material from the project(s) listed below 
+(Third Party Code).  Microsoft is not the original author of the Third Party Code.  Microsoft 
+reserves all other rights not expressly granted, whether by implication, estoppel or otherwise.
+
+1. fpm.h https://github.com/opensourcerouting/quagga/blob/master/fpm/fpm.h
```

Comment:
Need to indicate ISC license is selected.  How about
+1. fpm.h ISC License  https://github.com/opensourcerouting/quagga/blob/master/fpm/fpm.h